### PR TITLE
refactor: replace acc calculation in planning control modules

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

Replace acceleration calculation in each module to `/localization/acceleration`.

## Related links

https://github.com/tier4/autoware_launch/pull/398

## Tests performed

- [x] psim
- [x] scenario testing

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [ ] The PR has been properly tested.
- [ ] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.
- [ ] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by TakaHoribe** [comment]
cc. @kminoda

**Comment by TakaHoribe** [comment]
Must be merged after scenario testing

File: control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp [hunk 1]
```diff
@@ -185,9 +185,6 @@ class TRAJECTORY_FOLLOWER_PUBLIC PidLongitudinalController : public Longitudinal
   double m_ego_nearest_dist_threshold;
   double m_ego_nearest_yaw_threshold;
 
-  // 1st order lowpass filter for acceleration
-  std::shared_ptr<trajectory_follower::LowpassFilter1d> m_lpf_acc{nullptr};
-
```
Comment (takayuki5168) [inline control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: Do you think it's ok to remove the filter for acceleration? I guess whether the acceleration from the localization module may not be smooth enough for the controller.

Comment (kminoda) [reply control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: As long as it is OK for planning modules to use acceleration with unified smoothing parameter defined in localization/twist2accel, I guess planning modules do not need to filter acceleration by their own (Not sure if this assumption is correct or not, though).

Comment (takayuki5168) [reply control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: :sorehasou:

Comment (takayuki5168) [reply control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: @kminoda 
FYI.
As you may know, applying lowpass filter has pros/cons.
- pros: the value will be smooth.
- cons: the value will have a delay

How much planning/control modules wanna apply lowpass filter depends on how they wanna use the filtered value.
Therefore, in general, the filter gain should be tuned for each use-cases in each module.

Comment (TakaHoribe) [reply control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: Minoda-san investigated the LPF parameters in https://github.com/autowarefoundation/autoware.universe/pull/1089. I think taking a default LPF gain causes no problem.

Comment (takayuki5168) [reply control/trajectory_follower/include/trajectory_follower/pid_longitudinal_controller.hpp:190]: Got it. thanks.

**Comment by takayuki5168** [comment]
Lemme know if you finished modifying the PR.

**Comment by TakaHoribe** [comment]
@takayuki5168 checked in scenario_sim (780/823), which is the same as the main.  Ready for review.

**Comment by satoshi-ota** [comment]
@TakaHoribe Could you remove lowpass gain in `obstacle_stop_planner` ?

https://github.com/autowarefoundation/autoware.universe/blob/5e748e3f3bc2ae27eb9d917384a03504bf10fd2b/planning/obstacle_stop_planner/config/obstacle_stop_planner.param.yaml#L4

**Comment by TakaHoribe** [comment]
@satoshi-ota 

>Could you remove lowpass gain in obstacle_stop_planner ?

Done! And rebased now.

**Comment by satoshi-ota** [comment]
> @satoshi-ota
> 
> > Could you remove lowpass gain in obstacle_stop_planner ?
> 
> Done! And rebased now.

:god:

File: planning/obstacle_stop_planner/src/node.cpp [hunk 2]
```diff
@@ -223,31 +226,35 @@ void ObstacleStopPlannerNode::onTrigger(const Trajectory::ConstSharedPtr input_m
...
-        "waiting for obstacle pointcloud...");
+      waiting("obstacle pointcloud");
       return;
     }
 
-    if (!current_velocity_ptr_ && node_param_.enable_slow_down) {
-      RCLCPP_WARN_THROTTLE(
-        get_logger(), *get_clock(), std::chrono::milliseconds(1000).count(),
-        "waiting for current velocity...");
+    if (!current_velocity_ptr) {
```
Comment (satoshi-ota) [inline planning/obstacle_stop_planner/src/node.cpp:251]: @TakaHoribe special thanks!

**Review by satoshi-ota — APPROVED** [review]
LGTM

**Review by takayuki5168 — APPROVED** [review]
LGT

**Pull request merged**